gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/javaversions/BaselineJavaVersion.java

```diff
@@ -86,7 +86,17 @@ public void execute(CheckJavaVersionsTask task) {
                             task.getRuntimeVersion().set(extension.runtime());
                         }
                     });
-            project.getTasks().named("check").configure(check -> check.dependsOn(checkJavaVersions));
+
+            TaskProvider<CheckClasspathCompatible> checkRuntimeClasspathCompatible = project.getTasks()
+                    .register("checkRuntimeClasspathCompatible", CheckClasspathCompatible.class, task -> {
```

Comment:
Was originally going to do this for compileClasspath too, but I think I'd rather try and just roll out runtime first.